2024-12-01 15:24:23.436 | ERROR    | utils.ClientLogManager:log:49 - Loop Failed! Error: etcd connection failed, occurred on line 32 in file /home/turingzero/.local/lib/python3.10/site-packages/etcd3/client.py
2024-12-01 15:24:25.433 | ERROR    | utils.ClientLogManager:log:49 - Loop Failed! Error: etcd connection failed, occurred on line 32 in file /home/turingzero/.local/lib/python3.10/site-packages/etcd3/client.py
2024-12-01 15:24:27.434 | ERROR    | utils.ClientLogManager:log:49 - Loop Failed! Error: etcd connection failed, occurred on line 32 in file /home/turingzero/.local/lib/python3.10/site-packages/etcd3/client.py
2024-12-01 15:24:29.433 | ERROR    | utils.ClientLogManager:log:49 - Loop Failed! Error: etcd connection failed, occurred on line 32 in file /home/turingzero/.local/lib/python3.10/site-packages/etcd3/client.py
2024-12-01 15:24:31.434 | ERROR    | utils.ClientLogManager:log:49 - Loop Failed! Error: etcd connection failed, occurred on line 32 in file /home/turingzero/.local/lib/python3.10/site-packages/etcd3/client.py
2024-12-01 15:24:33.434 | ERROR    | utils.ClientLogManager:log:49 - Loop Failed! Error: etcd connection failed, occurred on line 32 in file /home/turingzero/.local/lib/python3.10/site-packages/etcd3/client.py
2024-12-01 15:24:35.434 | ERROR    | utils.ClientLogManager:log:49 - Loop Failed! Error: etcd connection failed, occurred on line 32 in file /home/turingzero/.local/lib/python3.10/site-packages/etcd3/client.py
2024-12-01 15:24:37.434 | ERROR    | utils.ClientLogManager:log:49 - Loop Failed! Error: etcd connection failed, occurred on line 32 in file /home/turingzero/.local/lib/python3.10/site-packages/etcd3/client.py
2024-12-01 15:24:39.434 | ERROR    | utils.ClientLogManager:log:49 - Loop Failed! Error: etcd connection failed, occurred on line 32 in file /home/turingzero/.local/lib/python3.10/site-packages/etcd3/client.py
2024-12-01 15:24:41.434 | ERROR    | utils.ClientLogManager:log:49 - Loop Failed! Error: etcd connection failed, occurred on line 32 in file /home/turingzero/.local/lib/python3.10/site-packages/etcd3/client.py
2024-12-01 15:24:43.434 | ERROR    | utils.ClientLogManager:log:49 - Loop Failed! Error: etcd connection failed, occurred on line 32 in file /home/turingzero/.local/lib/python3.10/site-packages/etcd3/client.py
2024-12-01 15:24:45.434 | ERROR    | utils.ClientLogManager:log:49 - Loop Failed! Error: etcd connection failed, occurred on line 32 in file /home/turingzero/.local/lib/python3.10/site-packages/etcd3/client.py
2024-12-01 15:24:47.425 | ERROR    | utils.ClientLogManager:log:49 - Loop Failed! Error: etcd connection failed, occurred on line 32 in file /home/turingzero/.local/lib/python3.10/site-packages/etcd3/client.py
2024-12-01 15:24:49.434 | ERROR    | utils.ClientLogManager:log:49 - Loop Failed! Error: etcd connection failed, occurred on line 32 in file /home/turingzero/.local/lib/python3.10/site-packages/etcd3/client.py
2024-12-01 15:24:51.434 | ERROR    | utils.ClientLogManager:log:49 - Loop Failed! Error: etcd connection failed, occurred on line 32 in file /home/turingzero/.local/lib/python3.10/site-packages/etcd3/client.py
2024-12-01 15:24:53.434 | ERROR    | utils.ClientLogManager:log:49 - Loop Failed! Error: etcd connection failed, occurred on line 32 in file /home/turingzero/.local/lib/python3.10/site-packages/etcd3/client.py
2024-12-01 15:24:54.165 | ERROR    | utils.ClientLogManager:log:49 - Loop Failed! Error: etcd connection failed, occurred on line 32 in file /home/turingzero/.local/lib/python3.10/site-packages/etcd3/client.py
2024-12-01 15:24:55.425 | ERROR    | utils.ClientLogManager:log:49 - Loop Failed! Error: etcd connection failed, occurred on line 32 in file /home/turingzero/.local/lib/python3.10/site-packages/etcd3/client.py
2024-12-01 15:24:55.434 | ERROR    | utils.ClientLogManager:log:49 - Loop Failed! Error: etcd connection failed, occurred on line 32 in file /home/turingzero/.local/lib/python3.10/site-packages/etcd3/client.py
2024-12-01 15:24:57.434 | ERROR    | utils.ClientLogManager:log:49 - Loop Failed! Error: etcd connection failed, occurred on line 32 in file /home/turingzero/.local/lib/python3.10/site-packages/etcd3/client.py
2024-12-01 15:24:58.164 | ERROR    | utils.ClientLogManager:log:49 - Loop Failed! Error: etcd connection failed, occurred on line 32 in file /home/turingzero/.local/lib/python3.10/site-packages/etcd3/client.py
